symfony/console/5.2/bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+if (!is_file(dirname(__DIR__).'/vendor/symfony/runtime/autoload.php')) {
+    throw new LogicException('Symfony Runtime is missing. Try running "composer require symfony/runtime".');
+}
+
+$kernel = require dirname(__DIR__).'/public/index.php';
+
+return function (array $context) use ($kernel) {
+    return new Application($kernel($context));
+};
```

Comment:
What about console only apps? It’s an edge case, but I had a friend recently build a symfony app without a public directory. It just looks weird to have the console depend on the front controller.

Reply:
They would create the Kernel inline as usual, right inside the close.
We could to the same here also, but I propose to reuse the definition in the index.php file here.

Reply:
I removed created the kernel from a require, it's now declared inline.